tests/e2e-autoscale/autoscale/cmd/verify/wait-and-validate-metrics.go

```diff
@@ -34,17 +34,13 @@ func main() {
 	var hpaName string
 	var timeout time.Duration
 	var numMetrics int
-	var kubeconfigPath string
 	var cpuValue int
 	var memoryValue int
 	var scaleDownWindow int
 	var scaleUpWindow int
 
-	defaultKubeconfigPath := filepath.Join(homedir.HomeDir(), ".kube", "config")
-
 	pflag.DurationVar(&timeout, "timeout", 5*time.Minute, "The timeout for the check.")
 	pflag.StringVar(&hpaName, "hpa", "", "HPA to check")
-	pflag.StringVar(&kubeconfigPath, "kubeconfig-path", defaultKubeconfigPath, "Absolute path to the KubeconfigPath file")
```

Comment:
do we not want this configurable anymore?

Reply:
It was not used by anyone before so...